src/main/resources/assets/paths/collectionsByRoleNameSummary.yaml

```diff
@@ -0,0 +1,34 @@
+get:
+  summary: Get DAR Collection Summaries By Role Name
+  description: |
+    Returns DAR collection summaries by Role Name
+      * Admin - all DAR collection summaries
+      * DAC Chair/Member - collections requesting datasets associated with their DAC 
+      * Signing Official - collections created by researchers belonging to their institution
+      * Researcher - collections created by the researcher 
+    Canceled DAR collections are not returned
```

Comment:
Which `Canceled` are we talking about here? If it's a researcher canceled collection then we have to show it since they can revise it as a draft on button click.